scripts/build_review_app.sh

```diff
@@ -34,29 +35,24 @@ hokusai review_app setup $NAME
 # --tag to name the image.
 # WARNING: This is likely going to take ~10 mins on your MBP.
 # Be patient and grab some baby carrots.
-hokusai registry push --force --skip-latest --verbose --tag $NAME
+hokusai registry push --force --skip-latest --overwrite --verbose --tag $NAME
 
 # Edit the K8S YAML to reference the proper Docker image
-# sed -i '' 's/:staging/:auct-281/g' hokusai/auct-281.yml
-#
-# The weird `-i ''` is needed on MacOS:
-# https://stackoverflow.com/a/44864004
-# WARNING: This might not work on Linux.
-sed -i '' "s/:staging/:$NAME/g" hokusai/$NAME.yml
+sed -i.bak "s/:staging/:$NAME/g" $review_app_file_path && rm $review_app_file_path.bak
 
 # Edit the K8S YAML to remove the instructions that enforce that the service
 # can only be accessible via Cloudflare
 #
 # First, remove the `loadBalancerSourceRanges:` line
-sed -i '' '/loadBalancer/d' hokusai/$NAME.yml
+sed -i.bak '/loadBalancer/d' $review_app_file_path && rm $review_app_file_path.bak
```

Comment:
Are we expecting to be able to leverage the recently introduced Jinja template support in hokusai to clean up some of these sed/regex string replacements? Perhaps in a follow-up iteration?

Reply:
> Are we expecting to be able to leverage the recently introduced Jinja template support in hokusai to clean up some of these sed/regex string replacements? Perhaps in a follow-up iteration?

I'm not aware of any plans to further parameterize these IPs, but that does seem like a good use-case and possible refactoring of the core hokusai setup here cc @izakp